react/Infos/index.jsx

```diff
@@ -71,7 +71,8 @@ Infos.propTypes = {
   icon: iconPropType,
   /** An important information will be displayed with red colors */
   isImportant: PropTypes.bool,
-  text: PropTypes.string,
+  /** Can be either a Text, or an object (for instance a ReactMardown component) */
+  text: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
I think `element` instead of `object` would be better  ? 

https://fr.reactjs.org/docs/typechecking-with-proptypes.html

```
  // Un élément React.
  optionalElement: PropTypes.element,
```

Reply:
thanks